apps/service/src/app/(all)/mypage/_components/UserProfile.tsx

```
@@ -160,6 +162,38 @@ const UserProfile = () => {
+          }
+        >
+          에러
+        </button>
+        <button
+          onClick={() => useToast.info('인포테스트 인포테스트 인포테스트 ')}
+        >
+          인포
+        </button>
+        <button
+          onClick={async () => {
+            const ok = await useConfirm('정말 탈퇴하시겠습니까?', '탈퇴');
```

Biome (2.4.6)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

(lint/correctness/useHookAtTopLevel)